stdlib/3/subprocess.pyi

```diff
@@ -2,19 +2,28 @@
 
 # Based on http://docs.python.org/3.6/library/subprocess.html
 import sys
-from typing import Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union, List, Type, Text
+from typing import (
+    Sequence, Any, Mapping, Callable, Tuple, IO, Optional, Union, List, Type, Text,
+    Generic, TypeVar,
+    overload,
+)
+from typing_extensions import Literal
```

Comment:
I think you can already import it from `typing`

Reply:
Apparently not yet?

Reply:
OK, I see, it is defined in `typing` stub under `if sys.version_info >= (3, 8): ...`